source/iostreams.tex

```diff
@@ -15161,8 +15161,8 @@
 \item The \grammarterm{root-name} element, if present.
 \item The \grammarterm{root-directory} element, if present.
 \begin{note}
-The generic format is required to ensure lexicographical
-comparison works correctly.
+It is possible that the use of the generic format is needed
+to ensure correct lexicographical comparison.
```

Comment:
Sigh. This would be so much easier to read as "The use of the generic format may be needed to ensure ..." if that wasn't a forbidden word.

Reply:
word :-(